src/assets/styles/abstracts/_breakpoints.scss

```diff
@@ -2,7 +2,7 @@
 	$min: breakpoint-value($name, $breakpoints);
 
 	@if $min {
-		@media (width >= rem($min)) {
+		@media (width > rem($min)) {
```

Comment:
Can you give me some context for this change?

Reply:
For the desired breakpoints, we need to use both breakpoint-up and breakpoint-down on lg breakpoint; however, both breakpoint rules includes lg with >= and <=, so breakpoint at exact point was not captured by both breakpoint-up and breakpoint-down, so the layout was off at the exact point. I changed breakpoint-up to not include specified breakpoint, so that can be handled under breakpoint-down.